### PR TITLE
fasttext has been relicensed to MIT in 2018

### PR DESCRIPTION
**Pull request by Anthchirp**
cf.
https://github.com/facebookresearch/fastText/commit/7842495a4d64c7a3bb4339d45d6e64321d002ed8

This also solves the issue of duplicate keys in the recipe which
prevents the automatic update bot running on this feedstock.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Anthchirp** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,7 +32,6 @@ about:
   summary: 'fastText - Library for efficient text classification and representation learning'
   dev_url: https://github.com/facebookresearch/fastText
   doc_url: https://fasttext.cc/docs/en/support.html
```
Comment (kylebgorman) [inline recipe/meta.yaml:34]: don't you need `license: MIT` or whatever here?

Comment (Anthchirp) [reply recipe/meta.yaml:34]: It's already there, in line 29

**Comment by kylebgorman** [comment]
Oh sorry, preview was hiding it from me.

**Pull request merged**